design/ybcluster_crd.yaml

```diff
@@ -0,0 +1,186 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: ybclusters.yugabytedb.yugabyte.com
```

Comment:
Not too happy with this resource name. What do other distributed databases use here?
cc @schoudhury

Reply:
How does `ybclusters.yugabyte.com` sound?
Some examples of other operator CR names:
Pod monitoring operator CR name by coreos.com: `podmonitors.monitoring.coreos.com`
Alert manager operator CR name by coreos.com: `alertmanagers.monitoring.coreos.com`
Istio control plane operator CR name by istio.io : `istiocontrolplanes.istio.io`

Reply:
Updated CR name to `ybclusters.yugabyte.com`. Marking as resolved.